javascript/javascript1/week1/app.js

```diff
@@ -0,0 +1,78 @@
+// Age-ify (A future age calculator)
+let yearOfBirth = 1987;
+let yearFuture = 2027;
+let age = yearFuture - yearOfBirth;
+console.log("You will be " + age + " years old in " + yearFuture);
+
+// Goodboy-Oldboy (A dog age calculator)
+let dogYearOfBirth = 2017;
+let dogYearFuture = 2027;
+let dogYear;
+let shouldShowResultInDogYears = true;
+let resultInHumanYears;
+if (dogYear) {
+    shouldShowResultInDogYears = (dogYearFuture - dogYearOfBirth) * 7;
+    console.log(
+        "Your dog will be " + shouldShowResultInDogYears + " dog years old in 2027"
+    );
+} else {
+    resultInHumanYears = dogYearFuture - dogYearOfBirth;
+
+    console.log(
+        "Your dog will be " + resultInHumanYears + " human years old in 2027"
+    );
+}
```

Comment:
OBS! 😊

Your approach is good and you're almost there! But, something is a little off in this solution. 

I have written a few questions below that might give you a hint as to what you should update and how: 
- What type of variable is shouldShowResultInDogYears? 
- Which variable do you want to use as the condition in the if else statement? 
- Which variable do you want to use for your calculation of the dog age? 

Don't hesitate to reach out on Slack if you want to talk it through. 💡
My Slack name is @Marie Louise Meyer 😊